pkg/etc/init.d/wavefront-proxy

```diff
@@ -63,6 +63,33 @@ if [[ -r $proxy_launch_conf ]]; then
 	fi
 fi
 
+if [[ ! -r $JAVA_HOME/bin/java ]]; then
```

Comment:
Few concerns here:

1) If JAVA_HOME is empty, i wonder if some users would have /bin/java, and if entering the loop in that case would be astonishing to them?
2) If their JAVA_HOME indicates a JRE that is specific to their deployment, but too old for wavefront (e.g. java7), then the customer will not be happy with this setup

What do you think of having something like WAVEFRONT_JAVA_HOME to indicate that the customer has opted into using their own JRE?